lib/tasks/load_data.rake

```diff
@@ -0,0 +1,36 @@
+require 'csv'
+namespace :load_data do
+  task :csvs => :environment do
+    puts "loading merchants"
+    CSV.foreach('db/data/merchants.csv', :headers => true, :encoding => 'ISO-8859-1:UTF-8') do |row|
+      Merchant.create([
+                      :name => row['name']
+                      ])
+    end
+    puts "loaded #{Merchant.all.count} merchants"
+
+    puts "loading transactions"
+    CSV.foreach('db/data/transactions.csv', :headers => true, :encoding => 'ISO-8859-1:UTF-8') do |row|
```

Comment:
@case-eee, in class, Josh suggested not importing or using data until you need it, so I tentatively avoided loading up all the CSV data unless explicitly outlined.

So, by that reasoning, if one CSV wasn't supposed to load up all the data, we'd need each one loaded specifically and independently. 

We're quickly leaning back towards loading it all up, and I think once we get there, we'll drop it down to a single "load everything" task.